test/specs/gfm/gfm.0.29.json

```diff
@@ -1,153 +1,147 @@
 [
   {
     "section": "Tables",
-    "html": "<table>\n<thead>\n<tr>\n<th>foo</th>\n<th>bar</th>\n</tr>\n</thead>\n<tbody>\n<tr>\n<td>baz</td>\n<td>bim</td>\n</tr></tbody></table>",
+    "html": "<table>\n<thead>\n<tr>\n<th>foo</th>\n<th>bar</th>\n</tr>\n</thead>\n<tbody>\n<tr>\n<td>baz</td>\n<td>bim</td>\n</tr>\n</tbody>\n</table>",
     "markdown": "| foo | bar |\n| --- | --- |\n| baz | bim |",
-    "example": 191
+    "example": "198"
```

Comment:
Does it matter if `example` is a string or a number?

Reply:
probably not but I fixed it so they are ints